backend/constants/messages.js

```
@@ -0,0 +1,83 @@
+  SUCCESS: {
+    USER_REGISTERED: 'User registered successfully',
+    LOGIN_SUCCESSFUL: 'Login successful',
+    PRODUCT_ADDED: 'Product added successfully',
+    NO_PRODUCTS_FOUND: 'No products found'
+  },
+  
+  // Error Messages
+  ERROR: {
+    // Authentication
+    INVALID_CREDENTIALS: 'Invalid email or password',
+    LOGIN_FAILED: 'Login failed',
```

Biome (2.4.4)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)